src/story_tests/reference_file_exists.robot

```diff
@@ -0,0 +1,53 @@
+*** Settings ***
+Resource         resource.robot
+Library          OperatingSystem
+Library          SeleniumLibrary
+Suite Setup      Open And Configure Browser
+Suite Teardown   Close Browser
+Test Setup       reset db
+
+*** Variables ***
+
+#vaihda latauskansion osoite koneelle josta testi tapahtuu
+#${DOWNLOAD_FOLDER}    /home/sadlaiho/Downloads
```

Comment:
could use the `HOME` environment variable or something like that to make this work better on different machines without having to manually change this variable

when it comes to grading, it is of course enough that the tests work locally